cupyx/tools/install_library.py

```diff
@@ -165,12 +165,17 @@ def install_lib(cuda, prefix, library):
         if library == 'cudnn':
             shutil.move(os.path.join(tmpdir, 'cuda'), destination)
         elif library == 'cutensor':
-            include = os.path.join(destination, 'include')
-            lib = os.path.join(destination, 'lib64')
-            shutil.move(os.path.join(tmpdir, 'libcutensor/include'), include)
-            if cuda.startswith('11'):
+            if cuda.startswith('11.'):
                 cuda = '11'
-            shutil.move(os.path.join(tmpdir, 'libcutensor/lib', cuda), lib)
+            shutil.move(
+                os.path.join(tmpdir, 'libcutensor', 'include'),
```

Comment:
Supper stupid and irrelevant.
Is there any strong reason not to use `Path` https://docs.python.org/3/library/pathlib.html
now that we only support python 3? I feel it is more readable than `os.path.join`.
But we don't need to change it now!

Reply:
Don't need to change it now, but I prefer `pathlib` too.

Reply:
Thanks for the suggestion! Agree with that, I'll rewrite it pathlib after merging this one and #4848.